editor/project_manager.cpp

```diff
@@ -663,123 +645,102 @@ void ProjectDialog::set_mode(Mode p_mode) {
 	mode = p_mode;
 }
 
+void ProjectDialog::set_project_name(const String &p_name) {
+	project_name->set_text(p_name);
+}
+
 void ProjectDialog::set_project_path(const String &p_path) {
 	project_path->set_text(p_path);
 }
 
 void ProjectDialog::ask_for_path_and_show() {
 	// Workaround: for the file selection dialog content to be rendered we need to show its parent dialog.
 	show_dialog();
-	_set_message("");
-
-	_browse_path();
+	_browse_project_path();
 }
 
 void ProjectDialog::show_dialog() {
+	// Set minimum sizes based on max autowrap line count. Label must be inside tree first.
 	if (mode == MODE_RENAME) {
+		msg->set_text("\n"); // One line
+	} else {
+		msg->set_text("\n\n"); // Two lines
+	}
+	msg->set_custom_minimum_size(msg->get_minimum_size());
+
```

Comment:
This ensures label never expands the window, but is hard coded...

![image](https://github.com/godotengine/godot/assets/14253836/18c4b2c8-8032-43b5-a2eb-5296344301ed)

Reply:
The message already has a minimum size limit to prevent the dialog from misbehaving. This should not be done, it only introduces instability.

Reply:
Is it okay that window is forced to resize when the initial 1 line turns into 2 lines.

Reply:
Why would we not want it to resize is the label now takes more space?